scripts/lib/CIME/case/case.py

```diff
@@ -1020,13 +1020,21 @@ def _create_caseroot_tools(self):
         except Exception as e:
             logger.warning("FAILED to set up exefiles: {}".format(str(e)))
 
-        # set up utility files in caseroot/Tools/
-        toolfiles = [os.path.join(toolsdir, "check_lockedfiles"),
-                     os.path.join(toolsdir, "getTiming"),
-                     os.path.join(toolsdir, "save_provenance"),
-                     os.path.join(toolsdir,"Makefile"),
-                     os.path.join(toolsdir,"mkSrcfiles"),
-                     os.path.join(toolsdir,"mkDepends")]
+        if "NEMO" in self._compsetname:
+            nemodir = os.path.join(self.get_value("CIMEROOT"),"..","components","nemo","NEMO36","bld")
```

Comment:
Why do you have a different mkDepends?  Do we really need it?

Reply:
I update the mkDepend to account for NEMO case